walrus/api.py

```diff
@@ -1,28 +1,31 @@
-import os.path
-import pandas as pd
+import os
+from walrus.FileConfig import FileConfig
```

Comment:
Module names should not be anything except snake_case (Class names should be CamelCase). I'm not sure why this class is called "FileConfig" though (there's no configuration happening here). Something simple like `Country` would do I think.

Reply:
Changed the class name and module name to Country in camel case and snake case format, respectively.